src/api/config/initializers/patch_pundit_exception.rb

```diff
@@ -0,0 +1,28 @@
+# This monkey patch allows custom messages for a Pundit error.
+#
+# This is an exact copy of Pundit #master.
+# Refs:
+#  - commit: https://github.com/varvet/pundit/commit/973b63b396c2a98099caf5eefd1c6841416eddfa
+#  - #master: https://github.com/varvet/pundit/blob/master/lib/pundit.rb#L24-L41
+#
+# TODO: Remove this after a new Pundit release.
+module Pundit
+  class NotAuthorizedError < Error
+    attr_reader :query, :record, :policy, :reason
+
+    def initialize(options = {})
+      if options.is_a? String # rubocop:disable Style/MethodCallWithArgsParentheses
```

Comment:
hey @intrip could you please drop this `rubocop:disable` and use the `option.is_a?(String)` ?